src/components/BookCard/BookCard.tsx

```diff
@@ -7,7 +11,22 @@ import type { Book } from '../../domain/books/types'
 type BookProps = Book;
 
 export const BookCard = (props: BookProps): JSX.Element => {
+  const dispatch: AppDispatch = useDispatch();
+
   return (
-    <div>Book Card</div>
-  )
-}
\ No newline at end of file
+    <div className="book-card">
+      <div className="book-card-img">
+        <img alt="Book Image" />
+      </div>
+      <div className="book-card-contents">
+        <p style={{ fontSize: "1.1em" }}>{props.name}</p>
+        <p style={{ fontSize: "25px" }}>{props.author}</p>
+        <p style={{ fontSize: "22px" }}>{props.description}</p>
+        <p style={{ fontSize: "20px" }}>${props.price}</p>
```

Comment:
These styles should have been defined in BookCard.css

Reply:
Fixed